src/main/java/de/hysky/skyblocker/utils/Location.java

```diff
@@ -121,4 +123,22 @@ public String id() {
     public static Location from(String id) {
         return Arrays.stream(Location.values()).filter(loc -> id.equals(loc.id())).findFirst().orElse(UNKNOWN);
     }
+
+    public static Location[] locationsWithout(Location... locations) {
+        ArrayList<Location> locationList = new ArrayList<>(List.of(values()));
+        locationList.removeAll(List.of(locations));
+        return locationList.toArray(new Location[0]);
+    }
+
+    private static Location[] hudLocations = null;
+
+	/**
+	 * @return All the locations available for the hud
+	 */
+    public static Location[] hudLocations() {
+        if (hudLocations == null) {
+            hudLocations = locationsWithout(UNKNOWN);
+        }
+        return hudLocations;
+    }
```

Comment:
This would leave UNKNOWN in a dropdown widget which is dumb since widgets don't show up in UNKNOWN

Reply:
In waypoints, it's for waypoints with invalid or unknown islands. I'm saying adding this here is more work than it's worth since leaving unknown in there is not a big deal.